### DISCUSSION TITLE
A Thank you.

### DISCUSSION THREAD

**Discussion by LindezaBlue** [new-sankaku/stable-diffusion-webui-simple-manga-maker#24] [General]
I've noticed there wasn't much going on in discussions, and I wanted to drop a message here and say _thank you_ to the Developers of this outstanding extension. I have been using this since it was first released and I love it!~

You are doing an outstanding job at keeping it updated, adding new features, and making such a wonderful tool!~ I really appreciate that you keep the main page updated and organized and simple to read/understand. 

Please keep up the outstanding work, and know you have a fan!~ :D

**Comment by new-sankaku** [comment]
Thanks for your kind words! We're glad you're enjoying the extension and we'll keep adding new features gradually. Your feedback is always welcome, and if you have any feature requests, feel free to post them in the Issues section. We'll try to implement them as we can. Keep in touch!
